service/storage_proxy.cc

```diff
@@ -3725,6 +3728,25 @@ class abstract_read_executor : public enable_shared_from_this<abstract_read_exec
     lw_shared_ptr<column_family>& get_cf() {
         return _cf;
     }
+
+    // Maximum latency of a successful request made to a replica (over all requests that finished up to this point),
+    // measured using a low resolution clock (10 ms granularity).
```

Comment:
Isn't 10ms granularity a bit too coarse for measuring latency?

Maybe it's better to keep using the `utils::latency_counter` - it uses a different implementation of the clock (`std::chrono::steady_clock`). I'm not sure if this clock has better or worse granularity, but it's probably better to stick to what was used before.

Just pointing that out, maybe someone more knowledgeable should comment.

Reply:
It is a bit too coarse. On the other hand steady_clock is too slow on some environments (virtualized).

Let's keep lowres_clock for now, I feel that it is a problem but let's not mix things.

Reply:
Ah, we use stead_clock now, so better to continue using steady_clock. In this case capturing the now() once becomes important.